### PR TITLE
Include Carthage Text

### PR DESCRIPTION
**Pull request by digoreis**
It's simple step-to-step for Carthage use in project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +31,25 @@ pod 'Valet'
 
 Or manually checkout the submodule with `git submodule add git@github.com:Square/Valet.git`, drag Valet.xcodeproj to your project, and add Valet as a build dependency.
```
Comment (dfed) [inline README.md]: Let's put this below Carthage under a subhead of `### Submodules`

File: README.md [hunk 2]
```diff
@@ -29,6 +31,25 @@ pod 'Valet'
...
+
+```bash
+$ brew update
+$ brew install carthage
+```
+
+To integrate Valet into your Xcode project using Carthage, specify it in your `Cartfile`:
+
+```ogdl
+github "Square/Valet" ~> 2.0.3
```
Comment (dfed) [inline README.md]: Do Carthage users need to be this specific? Could they say `~> 2.0`? I'd prefer not to prescribe a bug fix version in the README if that's not necessary.

**Comment by dfed** [comment]
Thanks for doing this! Fixes #54

**Comment by digoreis** [comment]
Hi Dan! Fix yours comments.

**Comment by dfed** [comment]
Thanks @digoreis!

**Pull request merged**